python/taichi/lang/glm.py

```diff
@@ -0,0 +1,92 @@
+import taichi as ti
+
+@ti.func
```

Comment:
Please test all the new functions. Some of them does not seem to work (e.g. `fract`)